internal/weaver/remoteweavelet.go

```diff
@@ -435,6 +435,7 @@ func (w *RemoteWeavelet) GetImpl(t reflect.Type) (any, error) {
 //
 // TODO(mwhittaker): Deduplicate with localweavelet.go.
 func (w *RemoteWeavelet) createComponent(ctx context.Context, reg *codegen.Registration) (any, error) {
+	w.syslogger.Debug("creating component", "component", reg.Name)
```

Comment:
Just checking whether these were for debugging or you intended to check them in!

Reply:
oops, my bad. Removed.